airflow/www/views.py

```diff
@@ -2025,12 +2029,14 @@ def _clear_dag_tis(
         ]
     )
     @action_logging
-    def clear(self):
-        """Clears the Dag."""
+    @provide_session
+    def clear(self, session: Session = NEW_SESSION):
```

Comment:
```suggestion
    def clear(self, *, session: Session = NEW_SESSION):
```

Reply:
Done, thanks.

Maybe I should update similar functions in this file to be consistent ? i.e `extra_links`, `rendered_k8s`,

Reply:
Maybe in a separate PR to not mix unrelated things.